code/modules/mob/living/death.dm

```diff
@@ -74,6 +74,17 @@
 
 	if (client)
 		client.move_delay = initial(client.move_delay)
+		if(!suiciding && !SSticker.first_death.len) //suicides don't clog up the list, only BRUTAL deaths here
+			var/list/L = list()
+			var/area/A = get_area(src)
+			L["name"] = name
+			L["role"] = null
+			if(mind.assigned_role)
+				L["role"] = mind.assigned_role
+			L["area"] = A.map_name
+			L["damage"] = "<font color='#FF5555'>[bruteloss]</font>/<font color='orange'>[fireloss]</font>/<font color='lightgreen'>[toxloss]</font>/<font color='lightblue'>[oxyloss]</font>/<font color='pink'>[cloneloss]</font>"
+			L["last_words"] = last_words
+			SSticker.first_death = L.Copy()

```

Comment:
why are you copying it? it doesn't go anywhere! just drop the ```.Copy()```

Reply:
1am code